demos/custom-tabs-example-app/src/main/java/org/chromium/customtabsdemos/NotificationParentActivity.java

```diff
@@ -69,8 +71,16 @@ public void onClick(View v) {
     }
 
     private void createAndShowNotification() {
+        NotificationManager notificationManager =
+                (NotificationManager) getSystemService(Context.NOTIFICATION_SERVICE);
+        NotificationChannel mChannel = notificationManager.getNotificationChannel(CT_NOTIFICATION_CHANNEL_ID);
```

Comment:
Don't we need to gate this behind a `Build.VERSION.SDK_INT` check, otherwise we'll crash on pre-O devices?

Reply:
Min SDK is currently set to 26 which already supports NotificationChannel.

Reply:
I was surprised as well 🤷 I'll check with Android DevRel whether there's a recommended min sdk for samples.